libraries/matrix/api/src/main/kotlin/io/element/android/libraries/matrix/api/room/MatrixRoom.kt

```diff
@@ -83,6 +84,8 @@ interface MatrixRoom : Closeable {
 
     suspend fun sendReaction(emoji: String, eventId: EventId): Result<Unit>
 
+    suspend fun forwardEvent(eventId: EventId, rooms: List<RoomId>): Result<Unit>
```

Comment:
See https://github.com/vector-im/element-x-android/pull/635#discussion_r1238063180. I was grouping the failures in a single exception type so we could later check it in the presenter and display a proper error for multi-selection. Since we canceled multi-selection for feature parity, right now it's not used.

Reply:
In any case, if this implementation seems difficult to use we can improve it, I'm just unsure what's the best DevX here.